medical_compliance/medical_compliance/api/tasks.py

```diff
@@ -74,6 +74,6 @@ def fetch_heart_rate_measurement():
         )
         heart_rate_measurement.save()
 
-    analyze_heart_rates.delay()
+    analyze_heart_rates.delay(last_meas)
```

Comment:
@rtud Nope, the local variable in ```fetch_heart_rate_measurement``` function is named ```last_meas```. Or maybe you want to say that we should name it ```last_measurement```?